.github/workflows/tests.yml

```diff
@@ -14,33 +14,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-
       - uses: ./.github/actions/setup-goversion
-
       - name: Build
         run: go build -v ./...
-
       - name: Lint
         uses: golangci/golangci-lint-action@971e284b6050e8a5849b72094c50ab08da042db8 # 6.1.1
         with:
           version: v1.56
-
       - name: Test
         run: go test -v ./...
-  check-for-doc-changes:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 #v4.2.2
       - name: Install make
         run: sudo apt-get update && sudo apt-get install -y make
         shell: bash
-      - name: Regenerate Helm Docs
+      - name: Check for Dashboards Drift
         run: |
-          make helm > /dev/null # we don't actually need to output, just the results
+          make build-dashboards > /dev/null
           if ! git diff --exit-code; then
-            echo "Helm docs are out of date. Please run 'make -C deploy/helm docs' and commit the changes."
+            echo "Dashboards are out of sync. Please run 'make build-dashboards' and commit the changes."
             exit 1
           fi
```

Comment:
#417 included changes in tests.yml that were duplicated so I could validate the file. GitHub Actions won't pick up new actions in PR's